pluto/build/edited.ini

```diff
@@ -1,4 +1,5 @@
 title = Edited
```

Comment:
I don't know what this "edited" sub-planet is for...was this just created during development? It doesn't seem to be in the menu on the home page.